projects/core/src/lib/utils/dom/get-focused.ts

```diff
@@ -3,7 +3,7 @@
  *
  * @return element or null
  */
-export function getFocused({activeElement}: Document): Element | null {
+export function maskitoGetFocused({activeElement}: Document): Element | null {
```

Comment:
Why do we expose it now?

Reply:
I need this utility in both `@maskito/core` & `@maskito/kit` packages (

Reply:
@waterplea review again, please